railties/lib/rails/generators/test_unit/system/templates/system_test_helper.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class ActionSystemTestCase < ActionSystemTest::Base
+  teardown do
+    take_failed_screenshot
+    Capybara.reset_sessions!
+  end
+end
```

Comment:
Great!

Should this call the Active Job helpers like `setup :set_queue_adapter_to_async`?